src/coreclr/vm/method.cpp

```diff
@@ -2107,19 +2107,26 @@ PCODE MethodDesc::TryGetMultiCallableAddrOfCode(CORINFO_ACCESS_FLAGS accessFlags
     if (accessFlags & CORINFO_ACCESS_LDFTN)
     {
         // Whenever we use LDFTN on shared-generic-code-which-requires-an-extra-parameter
-        // we need to give out the address of an instantiating stub.  This is why we give
-        // out GetStableEntryPoint() for the IsInstantiatingStub() case: this is
-        // safe.  But first we assert that we only use GetMultiCallableAddrOfCode on
+        // we need to give out the address of an instantiating stub. This is why we give
+        // out GetStableEntryPoint() for IsInstantiatingStub() via the IsWrapperStub() case: this is
+        // safe. But first we assert that we only use GetMultiCallableAddrOfCode on
         // the instantiating stubs and not on the shared code itself.
         _ASSERTE(!RequiresInstArg());
         _ASSERTE(!IsSharedByGenericMethodInstantiations());
 
-        // No other access flags are valid with CORINFO_ACCESS_LDFTN
-        _ASSERTE((accessFlags & ~CORINFO_ACCESS_LDFTN) == 0);
+        // Validate LDFTN flags are valid
```

Comment:
Is this necessary?

I would expect that `CORINFO_ACCESS_UNMANAGED_CALLER_MAYBE` flag would work fine with any other combination.

Reply:
I think I misunderstood the utility of these flags.